js/base/Exchange.js

```diff
@@ -472,6 +472,10 @@ module.exports = class Exchange {
         }
     }
 
+    print (... args) {
```

Comment:
No need to add it here, because it's already there: 

- https://github.com/ccxt/ccxt/blob/master/js/base/Exchange.js#L486

You can call it with `this.print (YOUR_ARGS_HERE)`